src/objdump.py

```diff
@@ -32,6 +32,7 @@ class ArchSettings:
     re_reg: Pattern[str]
     re_sprel: Pattern[str]
     re_includes_sp: Pattern[str]
+    re_reloc: str
```

Comment:
the name `re_reloc` indicates that it's a Pattern[str]

Reply:
good point, I'll fix